### ISSUE TITLE
Return class ids with dataloaders

### ISSUE THREAD

**Issue by crangelsmith** [alan-turing-institute/caked#15]
We need `train_y` and `val_y` returned as well as the pytorch dataloaders when doing ` loader.get_loader(batch_size=batch_s, split=splt)
` 

For reference is this lines (166 and 167):

https://github.com/alan-turing-institute/affinity-vae/blob/5c0d2c8da76dbbc39606822d8de2944248b74b61/avae/data.py#L166

**Comment by crangelsmith** [comment]
This is not necessary as we can obtain labels from the loaders. Thanks, @mooniean for helping debug! Closing.
